bedrock/firefox/templates/firefox/privacy/index.html

```diff
@@ -88,65 +62,61 @@ <h3 class="privacy-promise-sub-title">{{ ftl('firefox-privacy-hub-we-do-the-hard
-          {{ ftl('firefox-privacy-hub-theres-no-hidden-agenda-here-v2', privacy=url('privacy.notices.firefox')) }}
-        {% else %}
-          {{ ftl('firefox-privacy-hub-theres-no-hidden-agenda-here',
-                meetings='https://wiki.mozilla.org/WeeklyUpdates?utm_source=' ~ _entrypoint ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign ~ '&entrypoint=' ~ _entrypoint,
-                privacy=url('privacy.notices.firefox')) }}
-        {% endif %}
+        {{ ftl('firefox-privacy-hub-because-we-put-people-first', foundation='https://foundation.mozilla.org/?utm_source=' ~ _entrypoint ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign ~ '&entrypoint=' ~ _entrypoint) }}
       </p>
+      <p><a href="{{ url('mozorg.mission') }}">{{ ftl('firefox-privacy-hub-learn-more-about-our-mission') }}</a></p>
     {% endcall %}
-  </div>
 
-  <div class="privacy-promise-learn-more">
-      <ul class="mzp-l-content mzp-l-columns mzp-t-columns-two mzp-t-picto-center">
-      {% call picto(
-        title=ftl('firefox-privacy-hub-why-trust-firefox'),
-        body=True,
-        image=resp_img(
-          url='img/icons/mountain-purple.svg',
-          optional_attributes={
-            'class': 'mzp-c-picto-image',
-            'width': '150',
-            'loading': 'lazy'
-          }
-        ),
-        base_el='li'
-      ) %}
-        <p>
-          {{ ftl('firefox-privacy-hub-because-we-put-people-first',
-                 foundation='https://foundation.mozilla.org/?utm_source=' ~ _entrypoint ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign ~ '&entrypoint=' ~ _entrypoint) }}
-        </p>
-        <p><a href="{{ url('mozorg.mission') }}">{{ ftl('firefox-privacy-hub-learn-more-about-our-mission') }}</a></p>
-      {% endcall %}
-
-      {% call picto(
-        title=ftl('firefox-privacy-hub-your-privacy-by-the-product'),
-        body=True,
-        image=resp_img(
-          url='img/icons/privacy-shield.svg',
-          optional_attributes={
-            'class': 'mzp-c-picto-image',
-            'width': '150',
-            'loading': 'lazy'
-          }
-        ),
-        base_el='li'
-      ) %}
-        <p>{{ ftl('firefox-privacy-hub-firefox-products-work-differently') }}</p>
-        <p><a href="{{ url('firefox.privacy.products') }}">{{ ftl('firefox-privacy-hub-learn-about-our-products') }}</a></p>
-      {% endcall %}
-    </ul>
-  </div>
+    {% call picto(
+      title=ftl('firefox-privacy-hub-your-privacy-by-the-product'),
+      body=True,
+      image=resp_img(
+        url='img/icons/privacy-shield.svg',
+        optional_attributes={
+          'class': 'mzp-c-picto-image',
+          'width': '150',
+          'loading': 'lazy'
+        }
+      ),
+      base_el='li') %}
+      <p>{{ ftl('firefox-privacy-hub-firefox-products-work-differently') }}</p>
+      <p><a href="{{ url('firefox.privacy.products') }}">{{ ftl('firefox-privacy-hub-learn-about-our-products') }}</a></p>
```

Comment:
```suggestion
```

**Change:** Remove the link to the product page.

Reply:
(and its corresponding ftl string also)